Makefile

```diff
@@ -69,7 +69,7 @@ image: fmt operator-sdk ## Build the compliance-operator container image
 	$(GOPATH)/bin/operator-sdk build $(IMAGE_PATH) --image-builder $(RUNTIME)
 
 .PHONY: build
-build: fmt ## Build the compliance-operator binary
```

Comment:
Why was fmt removed?

Reply:
because for some reason fmt is not taking into account the GOFLAGS... and it keeps trying to use the modules coming from the sources instead of taking into account the vendor dir.

Reply:
cool, because that's exactly the issue I was seeing with my scapresults patch :)